### ISSUE TITLE
Should be able to pass custom parameters for tests

### ISSUE THREAD

**Issue by gfree76** [pester/AzureDevOpsExtension#61]
It would be good to be able to specify external data to be supplied to tests as documented here: https://pester.dev/docs/usage/data-driven-tests#providing-external-data-to-tests  This would allow us to pass the connection string to use to connect to a database, for example.

**Comment by ChrisLGardner** [comment]
I'd love to add support for this, it'll probably take me a while at present as I've got a lot of other things on. 

If you (or anyone) want to put in a PR for it then I'm happy to review it, if not then I'll try to get to it in the next few weeks but can't promise anything.

**Comment by gfree76** [comment]
Fair enough - I've only just started (last couple of weeks) using Azure DevOps or Pester, so it would probably take me a bit, but I'll clone the repo and see if I can figure out how to do it.

Thanks for taking a look!
g
________________________________
